packages/search-metadata-previews/src/snippet-editor/SnippetEditor.js

```diff
@@ -46,12 +46,15 @@ const CloseEditorButton = styled( SnippetEditorButton )`
 	margin-top: 24px;
 `;
 
+// The regex for the replacement variables we want to exclude from the SEO title before we measure the width.
+const excludedVars = new RegExp( "(%%sep%%|%%sitename%%)", "g" );
```

Comment:
This regex removes the separator and site title from the SEO title regardless of their position.